web-client/src/components/information/Further.vue

```diff
@@ -1,100 +1,73 @@
-                  }}
-                </ul>
-              </div>
-            </Card>
-            <Card custom-class="hover:text-primary dark:hover:text-light">
-              <div class="ml-10">
-                <ul class="text-lg ont-semibold text-gray-500 dark:text-light">
-                  <b>Foundation technology:</b>
-                  DISCO uses a public model-private data approach.
-                  The models are trained on the web app via
-                  <a
-                    class="text-blue-600"
-                    href="https://www.tensorflow.org/js"
-                  >TF.js</a>
-                  Decentralized training uses peer2peer communication (<a
-                    class="text-blue-600"
-                    href="https://peerjs.com/"
-                  >peerjs</a>).
-                </ul>
-              </div>
-            </Card>
-            <!--
-              To further seperate the text from the UI i18 interpolation can be be used:
-              https://kazupon.github.io/vue-i18n/guide/interpolation.html
-             -->
-
-            <Card custom-class="hover:text-primary dark:hover:text-light">
-              <div class="ml-10">
-                <ul class="text-lg ont-semibold text-gray-500 dark:text-light">
-                  <b>Research-focused design:</b>
-                  DISCO aims to enable decentralized training of
-                  machine learning algorithms, which is (i) efficient (<a
-                    class="text-blue-600"
-                    href="https://github.com/epfml/powergossip"
-                  >R1</a>,<a
-                    class="text-blue-600"
-                    href="https://github.com/epfml/ChocoSGD"
-                  >R2</a>), (ii) privacy-preserving (<a
-                    class="text-blue-600"
-                    href="https://arxiv.org/abs/2006.04747"
-                  >R3</a>), (iii) fault-tolerant and responsive (<a
-                    class="text-blue-600"
-                    href="https://arxiv.org/abs/1910.12308"
-                  >R4</a>), (iv) robust to malicious actors (<a
-                    class="text-blue-600"
-                    href="https://arxiv.org/abs/2012.10333"
-                  >R5</a>,<a
-                    class="text-blue-600"
-                    href="https://arxiv.org/abs/2006.09365"
-                  >R6</a>), while supporting (iv) fair participation, (v) transparency and (vi) interpretability.
-                </ul>
-              </div>
-            </Card>
-          </div>
+  <div class="flex justify-center">
+    <IconCard class="justify-self-center w-full lg:w-4/5">
+      <template #title>
+        Further Information
+      </template>
+      <template #content>
+        <div class="mb-4">
+          <b>Goal</b><br>
+          Disco enables collaborative and privacy-preserving training of machine learning models. Disco is an easy-to-use mobile app & web software, running directly in your browser.
```

Comment:
html needs some of its chars to be encoded :roll_eyes: 

```suggestion
          Disco enables collaborative and privacy-preserving training of machine learning models. Disco is an easy-to-use mobile app &amp; web software, running directly in your browser.
```